pkg/flare/envvars.go

```diff
@@ -34,6 +36,17 @@ var envvarNameWhitelist = []string{
 	"GODEBUG",
 	"GOMAXPROCS",
 	"GOTRACEBACK",
+
+	// Trace agent
+	"DD_APM_ENABLED",
+	"DD_APM_NON_LOCAL_TRAFFIC",
+
+	// Process agent
+	"DD_PROCESS_AGENT_ENABLED",
+}
+
+func init() {
+	envvarNameWhitelist = append(envvarNameWhitelist, config.ConfigEnvVars...)
```

Comment:
As the config bindings are setup in `pkg/config.init()` and golang does not guarantee us the execution order of the init functions, it might be safer to merge the lists when actually building the flare

Reply:
That's a good thing to consider 👍 I'll change it to future proof it.